apps/backend/user-service/src/main/resources/secrets.yml

```
@@ -0,0 +1,18 @@
+    jwt:
+      secret: alongrandomstringthatneedstobechangedmaybethisislongenough
+      expiration: 86400000
+    refresh:
+      expiration: 604800000
+  bcrypt:
+    strength: 12
+  mail:
+    host: smtp.gmail.com
+    port: 3087
+    username: rohitroy7014@gmail.com
+    password: blahblah
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)